bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
I explained the RHEL7 state in https://github.com/projectatomic/bubblewrap/pull/71#issuecomment-220990949

Again, what I'm objecting to is that `--unshare-user-try` does not actually work on RHEL7.  (And Debian stable, right?)

It's also unfortunate that RHEL7 fails with `EINVAL` and not `EPERM`...otherwise we could key off that.

Also, with `--unshare-user-try`, doesn't flatpak want to actually detect whether or not it worked?  Do you have logic for that now?